webapp/src/components/cardBadges.scss

```diff
@@ -1,12 +1,18 @@
 .CardBadges {
     display: flex;
     align-items: center;
+    margin-top: 4px;
+    height: 24px;
 
     span {
-        margin-right: 8px;
+        display: flex;
+        align-items: center;
+        margin-right: 4px;
 
         .Icon,
         .CompassIcon {
+            color: rgba(var(--center-channel-color-rgb), 0.64);
+            fill: currentColor;
```

Comment:
Hi @asaadmahmood, this `fill` is now also applied to `CheckIcon` and it looks like this:
![localhost_8088_bapwmhq8setns8xgjeabwcruizh_vfq1u7ms9efnbpb7k8epytj34sw](https://user-images.githubusercontent.com/5587620/201468633-16c7ae24-3305-4246-a090-b38c74130ab9.png)
before it looked like this:
![localhost_8088_bapwmhq8setns8xgjeabwcruizh_vfq1u7ms9efnbpb7k8epytj34sw (1)](https://user-images.githubusercontent.com/5587620/201468810-0eaa3479-1de6-4341-bfb4-bbc2076ee2db.png)

Is this a regression or expected change?

Reply:
@kamre Regression, I'll fix it.